tests/antares/services/local_services/test_study.py

```diff
@@ -1437,7 +1437,10 @@ def mock_error_in_sets_ini():
             raise CustomError("An error occurred while processing area can not be created")
 
         monkeypatch.setattr("antares.service.local_services.area_local._sets_ini_content", mock_error_in_sets_ini)
-        with pytest.raises(CustomError, match="An error occurred while processing area can not be created"):
+        with pytest.raises(
+            AreaCreationError,
+            match="Could not create the area test: An error occurred while processing area can not be created",
```

Comment:
Still not really clear, we don't have the area id and the msg isn't really in proper English